<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>串口:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QComboBox" name="portNameComboBox">
          <item>
           <property name="text">
            <string>COM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>COM25</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>波特率:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QComboBox" name="baudRateComboBox">
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>校验位:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QComboBox" name="parityComboBox">
          <item>
           <property name="text">
            <string>偶校验</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>奇校验</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>无校验</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>数据位:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QComboBox" name="dataBitsComboBox">
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>停止位:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QComboBox" name="stopBitsComboBox">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>清除接收</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>清除报文</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="4">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>瑞纳MODBUS表地址修改</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>原地址:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="3">
         <widget class="QLineEdit" name="oldaddr_lineEdit"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>新地址:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1" colspan="3">
         <widget class="QLineEdit" name="newaddr_lineEdit_2"/>
        </item>
        <item row="11" column="1" colspan="3">
         <widget class="QPushButton" name="revise_pushButton_3">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="openMyComBtn">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QPushButton" name="closeMyComBtn">
          <property name="text">
           <string>关闭串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLineEdit" name="sendMsgLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="sendMsgBtn">
          <property name="text">
           <string>抄表</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>MODBUS地址为LOOP2→Adr页面　</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>瑞纳MODBUS协议表地址</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>解析报文</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="textBrowser_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>数据接收</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTextEdit" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
